app/api/utils/handleError.js

```diff
@@ -79,6 +82,31 @@ const prettifyError = (error, { req = {}, uncaught = false } = {}) => {
   return result;
 };
 
+const getOriginalError = (data, error) => {
+  const original = data.original || error;
+
+  if (original instanceof Error) {
+    return original;
+  }
+
+  return null;
+};
+
+const sendLog = (data, error, errorOptions) => {
```

Comment:
I don't think we should fix this but, if you can think of a way to split this function, please let me know.